includes/namespace.php

```diff
@@ -176,6 +176,7 @@ function load_functions(): void {
  * @return array<string, array{body: string, headers: array<string,string>}|array<string, array{body: string, headers: array<string,string>}>> Modified reduce accumulator.
  */
 function rest_preload_api_request( array $memo, $path ): array {
+	// TODO, look at removing this PHP 5.2 support.
```

Comment:
This code is confusing, but we just copied this from core. But even there it doesn't really make sense, since PHP 5.6 is required for core.

"`array_reduce()` doesn't support passing an array in PHP 5.2" -> was this supposed to mean "`array_reduce()` doesn't support passing **a non-array** in PHP 5.2"? 🤔 

Anyway, given that we use an `array` parameter type hint, we can completely remove this if clause here.

Reply:
Should we do this in another PR or here?

Reply:
Here is fine